### DISCUSSION TITLE
Can Automa handle dialog box?

### DISCUSSION THREAD

**Discussion by louisfosse** [AutomaApp/automa#184] [Q&A] [answered]
Hello,

Just discovered the tool and it's amazing! 
However, I'm stuck with a window.alert where I have to click "OK". Can Automa handle those dialog boxes?

Thanks a lot :)

**Comment by Kholid060** [comment, marked as answer]
It's not possible for Automa to auto-click the ok button on the alert window. But instead, you can temporarily disable the alert window so it'll not show up when it's occurred. To do this, you can use the Javascript code block and then write below code

```js
const alertWindow = window.alert;
window.alert = () => {
  window.alert = alertWindow;
}

automaNextBlock();
```

**Reply by louisfosse** [reply]
Thanks a lot for your reply! it works perfectly.
